crowbar_framework/app/helpers/repositories_helper.rb

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2015, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module RepositoriesHelper
+  def td_not_available(required)
+    case required
+    when "mandatory"
+      "danger"
```

Comment:
I guess the idea here is to reuse an existing colour scheme provided by bootstrap?  Only problem is that it leads to counter-intuitive HTML, so there should be a comment explaining this, at least.  Other option is to create new CSS classes called `mandatory` etc. which match the existing colour scheme.  I think I probably prefer that, but maybe it's a matter of taste.